src/apps/colophon.py

```diff
@@ -0,0 +1,42 @@
+# TAXA March 2021
+
+import streamlit as st
+import base64
+from PIL import Image
+
+
+#st.set_page_config(layout="wide")
+
+
+#image = Image.open('image/TAXA.png')
+
+TAXA_LOGO = "image/TAXA.png"
+colophon = st.beta_container()
+
+#st.image(image, caption="Help discover species occurrences hiding in library collections.")
+
+with colophon:
+
+
+    st.markdown(
```

Comment:
I would remove this `st.markdown(..` call and replace with `st.image(image, caption="Help discover species occurrences hiding in library collections.")` that you have commented out in line 16.